internal/hostdbutil/db.go

```
@@ -65,6 +65,13 @@ func (db *EphemeralDB) ProcessChainRevertUpdate(cru *chain.RevertUpdate) error {
 	return nil
 }
 
+// Host returns information about a host.
+func (db *EphemeralDB) Host(hostKey types.PublicKey) hostdb.Host {
```

should return an error